tests/test_io.py

```diff
@@ -1039,5 +1312,55 @@ def test_can_read_all_files_in_magicc6_dir(file_to_read):
         mdata.read(MAGICC6_DIR, file_to_read)
 
 
+@pytest.mark.parametrize("file_to_read", [f for f in TEST_OUT_FILES])
+def test_can_read_all_valid_files_in_magicc6_out_dir(file_to_read):
+    if file_to_read.endswith(("PARAMETERS.OUT")):
+        read_cfg_file(join(TEST_OUT_DIR, file_to_read))
+    else:
+        # Check if the file is on the list of invalid files
+        for p in invalid_out_files:
+            if re.match(p, file_to_read):
+                # Return early as this file is marked as ignored.
+                return
+
+        mdata = MAGICCData(file_to_read)
+        mdata.read(TEST_OUT_DIR)
+
+
+@pytest.mark.parametrize("file_to_read", [f for f in TEST_OUT_FILES])
+def test_cant_read_all_invalid_files_in_magicc6_out_dir(file_to_read):
+    # Check if the file is on the list of invalid files
+    valid_filename = True
+    for p in invalid_out_files:
+        if re.match(p, file_to_read):
+            valid_filename = False
+    # Skip files not marked as invalid
+    if valid_filename:
+        return
+
+    mdata = MAGICCData(file_to_read)
+    try:
+        with pytest.raises(NoReaderWriterError):
+            mdata.read(TEST_OUT_DIR)
+    except InvalidTemporalResError:
+        pass
+
+
+@pytest.mark.parametrize("file_to_read", [f for f in listdir(TEST_OUT_DIR) if f.endswith('BINOUT') and f.startswith('DAT_')])
+def test_bin_and_ascii_equal(file_to_read):
+    try:
+        mdata_bin = MAGICCData(file_to_read)
+        mdata_bin.read(TEST_OUT_DIR)
+    except InvalidTemporalResError:
+        # Some BINOUT files are on a subannual time scale and cannot be read (yet)
+        return
+
+    mdata_ascii = MAGICCData(file_to_read.replace('BINOUT', 'OUT'))
+    mdata_ascii.read(TEST_OUT_DIR)
+
+    # There are some minor differences between in the dataframes due to availability of metadata in BINOUT files
+    mdata_ascii.df.columns = mdata_ascii.df.columns.droplevel('UNITS').droplevel('TODO')
```

Comment:
change this so that the binary files come back with units unknown and TODO equal to SET